lib/include/openamp/virtio.h

```diff
@@ -101,6 +101,8 @@ __deprecated static inline int deprecated_virtio_dev_slave(void)
 #warning "VIRTIO_DEVICE_ONLY is deprecated, please use VIRTIO_DRIVER_SUPPORT=0"
 #endif
 
+#define IS_ENABLED(option) (option == 1)
```

Comment:
Seems that this creates a redefinition issue in Zephyr context.
Perhaps this define should be in a separate include that only been includes by open_amp  source file.
What about adding a include/internal/utilities.h file for that?

Reply:
Seems that doesn't work either, if any source file includes the new utilities.h but also is built in the Zephyr context which also has IS_ENABLED() defined due to inclusions from `libmetal`, we get the same issue.

Simple solution is to change this to a unique name, since this is for checking if `virtio` options are enabled, I'll just rename it to `VIRTIO_ENABLED`.

Reply:
Fair enough, if we need a more generic macro for some other config we will address that later.